core/src/test/scala/unit/kafka/zk/KafkaZkClientTest.scala

```diff
@@ -276,6 +301,49 @@ class KafkaZkClientTest extends ZooKeeperTestHarness {
 
     zkClient.deleteAclChangeNotifications()
     assertTrue(zkClient.getChildren(AclChangeNotificationZNode.path).isEmpty)
+  }
+
+  @Test
+  def testDeleteTopicPathMethods() {
+    val topic1 = "topic1"
+    val topic2 = "topic2"
+
+    assertFalse(zkClient.isTopicMarkedForDeletion(topic1))
+    assertTrue(zkClient.getTopicDeletions.isEmpty)
+
+    zkClient.createDeleteTopicPath(topic1)
+    zkClient.createDeleteTopicPath(topic2)
+
+    assertTrue(zkClient.isTopicMarkedForDeletion(topic1))
+    assertEquals(Seq(topic1, topic2), zkClient.getTopicDeletions)
+
+    zkClient.deleteTopicDeletions(Seq(topic1, topic2))
+    assertTrue(zkClient.getTopicDeletions.isEmpty)
+  }
+
+  @Test
+  def testEntityConfigManagementMethods() {
```

Comment:
Saw the following unit test failure. 

kafka.zk.KafkaZkClientTest > testEntityConfigManagementMethods FAILED
    java.lang.AssertionError: expected:<List(topic1, topic2)> but was:<Buffer(topic2, topic1)>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:118)
        at org.junit.Assert.assertEquals(Assert.java:144)
        at kafka.zk.KafkaZkClientTest.testEntityConfigManagementMethods(KafkaZkClientTest.scala:344)